metricbeat/module/kubernetes/state_daemonset/state_daemonset.go

```diff
@@ -89,11 +89,7 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 	return &MetricSet{
 		BaseMetricSet: base,
 		prometheus:    prometheus,
-<<<<<<< HEAD
-		enricher:      util.NewResourceMetadataEnricher(base, &kubernetes.ReplicaSet{}, mod.GetPerfMetricsCache(), false),
-=======
-		enricher:      util.NewResourceMetadataEnricher(base, &kubernetes.DaemonSet{}, mod.GetMetricsRepo(), false),
->>>>>>> 5503761995 (Feature/remove k8s cache (#32539))
+		enricher:      util.NewResourceMetadataEnricher(base, &kubernetes.ReplicaSet{}, mod.GetMetricsRepo(), false),
```

Comment:
strange that we have here `ReplicaSet`, but it is the same as is in 7.17 - nothing introduced here on this regard